src/Raven.Server/Documents/Indexes/Workers/Cleanup/CleanupTimeSeries.cs

```diff
@@ -75,7 +75,7 @@ protected override void HandleDelete(TombstoneIndexItem tombstone, string collec
         protected virtual void HandleTimeSeriesDelete(TombstoneIndexItem tombstone, string collection, Lazy<IndexWriteOperationBase> writer,
             QueryOperationContext queryContext, TransactionOperationContext indexContext, IndexingStatsScope stats)
         {
-            writer.Value.DeleteTimeSeries(tombstone.LowerId, tombstone.LuceneKey, stats);
+            writer.Value.DeleteByPrefix(tombstone.LuceneKey, stats);
```

Comment:
This is not Lucene / Corax specific code but we use `.LuceneKey`. Is it intentional?

Reply:
done - changed to `PrefixKey `